makefile

```diff
@@ -10,7 +10,11 @@ sysconfdir=/etc
 wondershaper:
 	exit;
   
-install:
+
+wondershaper.service: wondershaper.service.in
+	sed 's|@sbindir@|$(sbindir)|' wondershaper.service.in > wondershaper.service
```

Comment:
This seems pretty convoluted. I understand you're trying to be flexible, but wouldn't anyone who wants a different directory be able to edit both the makefile and the services file.

Can you point to any convention/precedent of this procedure with the ```.in``` and the usage of sed within the make?